src/view.c

```diff
@@ -772,6 +772,17 @@ view_discover_output(struct view *view)
 	view->output = view_output(view);
 }
 
+void
+view_evacuate_region(struct view *view)
+{
+	assert(view);
+	if (view->tiled_region) {
+		assert(!view->tiled_region_evacuate);
```

Comment:
Hmm, I found two places where `tiled_region` is set (other than to `NULL`):
- https://github.com/labwc/labwc/blob/0ca6c4c763eaee091a3abeddd135eb6272fcfd26/src/view.c#L369
- https://github.com/labwc/labwc/blob/0ca6c4c763eaee091a3abeddd135eb6272fcfd26/src/view.c#L950

In both cases it is accompanied by `zfree(view->tiled_region_evacuate)`. So it appeared to me impossible for both `tiled_region` and `tiled_region_evacuate` to be non-null at the same time. Thus `assert(!view->tiled_region_evacuate)` seemed safe.

However maybe it would be best to be safe/flexible here. What about this?
```
	assert(view);
	if (view->tiled_region && !view->tiled_region_evacuate) {
		view->tiled_region_evacuate = xstrdup(view->tiled_region->name);
	}
	view->tiled_region = NULL;
```

Reply:
IMHO this function should really *never* be called if there is no region to evacuate.
So not using an assert for `view->tiled_region` allows hiding future bugs in the call path.

Regarding `tiled_region_evacuate`: It might be true that the check is redundant, however, its a logic change from the state before and I am too tired right now to look through all possible call chains and conditions to verify that it will never be set along with `tiled_region` itself.

Reply:
All right, I updated the `assert`s as you suggested.